src/ngSanitize/sanitize.js

```diff
@@ -262,11 +289,30 @@ function makeMap(str, lowercaseKeys) {
   return obj;
 }
 
+var inertBodyElement;
+(function(window) {
+  var doc;
+  if (window.document && window.document.implementation) {
+    doc = window.document.implementation.createHTMLDocument("inert");
+  } else {
+    throw $sanitizeMinErr('noinert', "Can't create an inert html document");
+  }
+  var docElement = doc.documentElement || doc.getDocumentElement();
+  var bodyElements = docElement.getElementsByTagName('body');
+  if (bodyElements.length === 1) {
```

Comment:
usually there should be only one body, but in IE there is none, so we need to create it. I'll add a comment to that code